doc/titus-v3-spec.md

```diff
@@ -1102,6 +1105,43 @@ An owner of a job.
 
 
 
+<a name="com.netflix.titus.PlatformSidecar"></a>
+
+### PlatformSidecar
+Definition of a request to add a platform sidecar alongside a task
+Note that this is *not* a user-defined sidecar, that is why it just has a
+name. These platform-sidecars are attached a task start time, and the
+definition of what the sidecar is is not baked into the job itself, just the
+intent.
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| name | [string](#string) |  | (Required) Name of the platform sidecar requested |
+| channel | [string](#string) |  | (Optional) Channel representing a pointer to releases of the sidecar |
+| constraints | [PlatformSidecar.ConstraintsEntry](#com.netflix.titus.PlatformSidecar.ConstraintsEntry) | repeated | (Optional) Arguments, KV pairs for configuring the sidecar |
+
+
+
+
+
+
+<a name="com.netflix.titus.PlatformSidecar.ConstraintsEntry"></a>
+
+### PlatformSidecar.ConstraintsEntry
```

Comment:
I'm guessing this Markdown is generated from the protos, which is slick 😎 But it would be helpful to have examples or more descriptions here, since I don't think the name is very clear for consumers without some additional context.

Reply:
Totally.

Once we have some sort of actual working mutator, I'll (`HelloWorldSidecar` ?) or some sort of example thing that will always be around, I'll copy that into here so users have real-world working examples.